helper/schema/resource.go

```diff
@@ -45,17 +46,8 @@ type Resource struct {
 	// their Versioning at any integer >= 1
 	SchemaVersion int
 
-	// LegacySchema is a record of the last schema version and type that
-	// existed before the addition of an UpgradeState function.
-	//
-	// This allows the resource schema to continue to evolve, while providing a
-	// record of how to decode a legacy state to be upgraded.
-	//
-	// LegacySchema is required when implementing UpgradeState.
-	LegacySchema LegacySchemaVersion
-
 	// MigrateState is deprecated and any new changes to a resource's schema
-	// should be handled by UpgradeState. Existing MigrateState implementations
+	// should be handled by StateUpgraders. Existing MigrateState implementations
```

Comment:
At the end of this comment it still talks about `LegacySchema.Version`. I guess that should now be "lower than the first version specified in `StateUpgraders`?

Reply:
thanks, missed that one.